demos/code/code.js

```diff
@@ -93,7 +93,7 @@ Code.workspace = null;
  * Extracts a parameter from the URL.
  * If the parameter is absent default_value is returned.
  * @param {string} name The name of the parameter.
- * @param {string} defaultValue Value to return if parameter not found.
+ * @param {string} defaultValue Value to return if paramater not found.
```

Comment:
Revert this misspelling. The original is correct. Compare with [Wikipedia](https://en.wikipedia.org/wiki/Parameter_(computer_programming))